### ISSUE TITLE
Should we add PUT/POST endpoints for tables?

### ISSUE THREAD

**Issue by friendtocephalopods** [amundsen-io/amundsen#36]
We wanted to do this at Square so our databuilder instance doesn't have to connect directly to the neo4j database instance. It can just call some simple metadata library APIs that handle the database details.

We think this gives a better separation of concerns between the databuilder and the metadata library code.

I built this for the Square instance of amundsen, I'd be happy to upstream it if y'all are amenable.

**Comment by jornh** [comment]
Maybe not a priority yet, but eventually if you want to federate between Amundsen and other metadata stores https://www.odpi.org/blog/2019/07/08/implementing-an-open-metadata-connector seems worth a look.

I believe i saw ING is part of the ODPi an Egeria consortium as well @

There of course can be different APIs in parallel. But why reinvent the wheel unless it’s a better or simper one ☝️
